src/config_reader.rs

```diff
@@ -89,10 +90,15 @@ fn get_list_of_i3bindings_from_content(
                 "bindcode" => "Code",
                 _ => "Unknown",
             };
+            let _run_type = match caps.name("type") {
```

Comment:
suggestion: rename this to `run_type`. Leading underscore is not necessary since it's not a throwaway variable.

```suggestion
            let run_type = match caps.name("type") {
```

Reply:
add changes on latest commit